nemo/collections/asr/models/hybrid_rnnt_ctc_bpe_models.py

```diff
@@ -144,7 +144,7 @@ def _setup_dataloader_from_config(self, config: Optional[Dict]):
             return dataset
 
         shuffle = config['shuffle']
-        if config.get('is_tarred', False):
+        if isinstance(dataset, torch.utils.data.IterableDataset):
```

Comment:
++

Reply:
Dataset cannot be None. There is the following code just above the line you have highlighted:

```python
if dataset is None:
    return None
```